scripts/hudson/narayana.bat

```diff
@@ -0,0 +1,75 @@
+if not defined WORKSPACE (call:fail_build & exit -1)
+
+set NOPAUSE=true
+@echo off
+
+call:comment_on_pull "Started testing this pull request on Windows: %BUILD_URL%"
+
+java -version 2>tmp.txt
+findstr version tmp.txt > version.txt
+FOR /F delims^=^"^ tokens^=2 %%i in (version.txt) DO @set JAVA_VERSION=%%i
+if %JAVA_VERSION% == 9 set MAVEN_OPTS="--add-modules=java.corba,java.xml.bind,java.xml.ws"
+
+IF NOT [%NOTMAIN%] == [] (call:build_spi_pr
+call build.bat clean install %* || (call:comment_on_pull "Tests failed on Windows - Narayana Failed %BUILD_URL%" & exit -1)
+)
+
+IF NOT [%NOTBT%] == [] (echo "set WILDFLY_MASTER_VERSION=%WILDFLY_MASTER_VERSION% set JBOSSAS_IP_ADDR=%JBOSSAS_IP_ADDR%"
+rem SHUTDOWN ANY PREVIOUS BUILD REMNANTS
+FOR /F "usebackq tokens=5" %%i in (`"netstat -ano|findstr 9990.*LISTENING"`) DO taskkill /F /PID %%i
+tasklist
+taskkill /F /IM mspdbsrv.exe
+taskkill /F /IM testsuite.exe
+taskkill /F /IM server.exe
+taskkill /F /IM client.exe
+taskkill /F /IM cs.exe
+wmic Path win32_process Where "Caption Like '%java.exe%' AND CommandLine Like '%standalone%'" Call Terminate
+tasklist
+
+
+echo "Finished build"
+
+call:comment_on_pull "Tests passed on Windows - Job complete %BUILD_URL%"
+
+rem ------------------------------------------------------
+rem -                 Functions below                    -
+rem ------------------------------------------------------
+
+goto:eof
+
+:build_spi_pr
+  if "%SPI_BRANCH%"=="" goto:eof
+  
+  echo "Cloning SPI"
+  rmdir /S /Q jboss-transaction-spi
+  git clone https://github.com/jbosstm/jboss-transaction-spi.git || (call:comment_on_pull "SPI clone Failed %BUILD_URL%" & exit -1)
+  cd jboss-transaction-spi
+  if defined SPI_BRANCH git fetch origin +refs/pull/*/head:refs/remotes/jbosstm/pull/*/head & git checkout %SPI_BRANCH%
+  if %ERRORLEVEL% NEQ 0 exit -1
+  cd ..
+  call build.bat clean install "-f" "jboss-transaction-spi\pom.xml" || (call:comment_on_pull "SPI build Failed %BUILD_URL%" & exit -1)
+goto:eof
+
+:fail_build
+  call:comment_on_pull "Build failed %BUILD_URL%"
+  tasklist & FOR /F "usebackq tokens=5" %%i in (`"netstat -ano|findstr 9990.*LISTENING"`) DO taskkill /F /PID %%i
+  tasklist
```

Comment:
I am reasonably sure that all of the exe's in the tasklist are blacktie executables.

Reply:
I will remove them, thanks